index.html

```diff
@@ -169,20 +228,20 @@ <h1 class="about-me-title">About me</h1>
           </li>
         </ul>
       </form>
-      <ul class="nobull links">
-        <li>
-          <a class="social fa-brands fa-twitter" href="http://twitter.com/sps676"></a>
-        </li>
-        <li>
-          <a class="social fa-brands fa-linkedin" href="https://www.linkedin.com/in/sergiy-shkolnik-7801a53/"></a>
-        </li>
-        <li>
-          <a class="social fa-brands fa-github" href="https://github.com/SergSkol"></a>
-        </li>
-      </ul>
-      <section class="email">
-        <p>sps676@gmail.com</p>
-      </section>
+    </section>
+    <ul class="nobull links">
+      <li>
+        <a class="social fa-brands fa-twitter" href="http://twitter.com/sps676"></a>
+      </li>
+      <li>
+        <a class="social fa-brands fa-linkedin" href="https://www.linkedin.com/in/sergiy-shkolnik-7801a53/"></a>
+      </li>
+      <li>
+        <a class="social fa-brands fa-github" href="https://github.com/SergSkol"></a>
+      </li>
+    </ul>
```

Comment:
- [ ] nice adding asked changes but you forgot to add a gray line above your social media icons as in the Figma design so please try to add in order to match The Figma design 
- Yours
<img width="1420" alt="Screen Shot 2022-10-05 at 15 38 20" src="https://user-images.githubusercontent.com/61790587/194088300-af9dd9cc-57fb-444e-8c66-661a263a953d.png">
-Figma
<img width="807" alt="Screen Shot 2022-10-05 at 15 38 49" src="https://user-images.githubusercontent.com/61790587/194088325-c3698c0f-eea6-4bad-8d5f-55d9e8fa140e.png">